sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -152,4 +264,31 @@ public CompletionPostResponse messagesHistory() {
 
     return API.orchestrationV1EndpointsCreate(config);
   }
+
+  /**
+   * Chat request to OpenAI through the Orchestration service with an anonymization masking template
+   *
+   * @return the assistant message response
+   */
+  @GetMapping("/maskingAnonymization")
+  @Nonnull
+  public CompletionPostResponse maskingAnonymization() {
+    final var config = MASKING_CONFIG.apply(MaskingProviderConfig.MethodEnum.ANONYMIZATION);
+
+    return API.orchestrationV1EndpointsCreate(config);
+  }
+
+  /**
+   * Chat request to OpenAI through the Orchestration service with a pseudonymization masking
+   * template
+   *
+   * @return the assistant message response
+   */
+  @GetMapping("/maskingPseudonymization")
```

Comment:
```diff
- @GetMapping("/maskingPseudonymization")
+ @GetMapping("/maskingPseudoAnonymization")
```

Reply:
Why? Orchestration calls it `pseudonymization`.